### DISCUSSION TITLE
What is the time used in the content assistant response?

### DISCUSSION THREAD

**Discussion by jravenel** [jupyter-naas/abi#146] [Q&A] [answered]
When I ask for the time of the post, it gives a time I'm not sure what the reference is (GMT?) is there something we can do to show the time of the post based on user preferences? or maybe have one reference timezone in the text displayed?

**Comment by FlorentRvl** [comment, marked as answer]
> When I ask for the time of the post, it gives a time I'm not sure what the reference is (GMT?) is there something we can do to show the time of the post based on user preferences? or maybe have one reference timezone in the text displayed?

The time of the post is related to user location on LinkedIn.

<img width="886" alt="image" src="https://github.com/jupyter-naas/abi/assets/48032461/c8a25c75-2763-4f12-aa53-36044239b27c">
